### DISCUSSION TITLE
ps2 pcb Shell print

### DISCUSSION THREAD

**Discussion by dyf2007** [pmgducati/Blue-Retro-AIO-Units#7] [Q&A] [answered]
I tested the ps 2 successfully, this is my own painting of the shell, not very beautiful.
[Uploading sn74ahct125.pdf…]
![IMG_20220403_213123](https://user-images.githubusercontent.com/84038001/162227399-e04c7c34-cf4d-4ad6-a310-66d7c675ebfd.jpg)
![78ab2155ddf70a9c](https://user-images.githubusercontent.com/84038001/162227413-c79cfafd-c9f0-410f-b40b-7dd5bfd83272.jpg)

**Comment by pmgducati** [comment, marked as answer]
Nice Work!

**Reply by dyf2007** [reply]
![-1c7ee1f86c568a1](https://user-images.githubusercontent.com/84038001/164432840-8e9cbf1a-7e5e-4809-bed6-d864ba6e24d0.png)
![-6ce619720903a02c](https://user-images.githubusercontent.com/84038001/164432911-f1dcc896-9549-4651-b397-7fb28dfad28b.png)
![2e456ea6a889fbed](https://user-images.githubusercontent.com/84038001/164432980-b314086c-bde6-4f56-9f7c-d90a58d40b6c.png)
![mmexport1650016744094](https://user-images.githubusercontent.com/84038001/164433054-ab2ff329-72c2-4699-a710-9a6ac218ba6d.png)
I redraw some PCB, I do not have so many machines, can only find friends to test.
![IMG_20220419_050327](https://user-images.githubusercontent.com/84038001/164433515-761a6e5a-52b4-4793-b960-a5dde0cfcd29.jpg)

**Reply by dyf2007** [reply]
Redrawing Blue-Retro worked.
